app/assets/stylesheets/application.scss

```diff
@@ -11,7 +11,7 @@
 @import "patternfly-sprockets";
 @import "patternfly";
 @import "mixin";
-@import "/**/*";
+@import "/*";
```

Comment:
`@import "/**/*"` imports all files in dir including subdir. In that case the `email.css` is included to `application.scss`, and caused the integration errors. 
`@import "/*"` imports all files in dir except subdir files.  So I created new subdir in stylesheets for all un-imported files.